app/views/alaveteli_pro/subscriptions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>AlaveteliPro::Subscriptions#show</h1>
+<p>Find me in app/views/alaveteli_pro/subscriptions/show.html.erb</p>
+
+<h2><%= _('Your current subscription') %></h2>
+
+<%= render partial: 'subscription',
+           locals: { subscription: @customer.subscriptions.first } %>
+
+<h2><%= _('Your payment details') %></h2>
+
+<%= render partial: 'card', locals: { card: @customer.sources.first } %>
```

Comment:
Don't we want [`@customer.default_source`](https://stripe.com/docs/api/ruby#customer_object-default_source) here?

Reply:
`@customer.default_source` is just the card id, have done it a bit differently and assigned `@card` in the controller